### PR TITLE
Connect main components

### PR DESCRIPTION
**Pull request by Gabrieldowen**

### CODE REVIEW

File: ParseNOTAM.py [hunk 1]
```diff
@@ -10,15 +10,14 @@ def ParseNOTAM(json_data = None):
 
         # Reading JSON data from the file
         with open(file_path, 'r') as json_file:
-            json_data = json.load(json_file)
+            json_data_test = json.load(json_file)
```
Comment (n-grounds) [inline ParseNOTAM.py:13]: seems odd you are changing the variable name here from `json_data` to `json_data_test` when the code below still refers to `json_data`

File: ParseNOTAM.py [hunk 2]
```diff
@@ -10,15 +10,14 @@ def ParseNOTAM(json_data = None):
...
-            json_data = json.load(json_file)
+            json_data_test = json.load(json_file)
 
-
-    # create a class for each NOTAM
     NOTAMs = []
-    for item in json_data['items']:
-        NOTAMs.append(Models.Notam(item['properties']['coreNOTAMData']['notam']))
-        
+    for notam in json_data:
```
Comment (n-grounds) [inline ParseNOTAM.py:16]: seems odd to loop over every field in `json_data` and then for each one loop over over child in its `items` field ... perhaps a comment here about the expected structure of the JSON data (or even a short example) is warranted?

File: templates/index.html [hunk 3]
```diff
@@ -6,6 +6,15 @@
     <h1>Welcome to My Flask App</h1>
       <div class="container mt-5">
         <form method="POST">
+            <div class="mb-3">
+                <label for="startAirport" class="form-label">Start airport</label>
+                <input type="text" class="form-control" id="responseFormat" name="startAirport">
+            </div>
+            <div class="mb-3">
+                <label for="destAirport" class="form-label">dest airport</label>
```
Comment (n-grounds) [inline templates/index.html:14]: For consistency shouldn't the label value be "Dest airport"?

File: MinimalCirclesPath.py [hunk 4]
```diff
@@ -67,20 +67,26 @@ def getPath( startLat, startLong, destLat, destLong, radius, pathWidth):
...
-    # list of coordinates returned to call API with
-    coordList = [(startLat, startLong)]
+    
       
     # gets total distance from start to finish
     totalDistance = getDistance(startLat, startLong, destLat, destLong)
 
     # gets step distance (pythagorean theorum)
-    stepDistance = sqrt((radius)**2-(pathWidth/2)**2)
+    stepDistance = 2 * sqrt((radius)**2-(pathWidth/2)**2)
```
Comment (n-grounds) [inline MinimalCirclesPath.py:76]: obviously some kind of bug or problem was found which required this change ... are there unit tests for this function?  do we have an example start/dest for which the intermediate points this function calculates are known a priori and verified?

this is exactly the kind of complex code (coding a complex algorithm) for which unit tests are ideally suited, the implementation of this algorithm has to be "right" or the usefulness of this whole program becomes effectively zero

**Review by JmzCS50 — APPROVED** [review]
LG

File: MinimalCirclesPath.py (hunk 4 above)
Comment (Gabrieldowen) [reply MinimalCirclesPath.py:76]: I plan on making unit test that's next on my todo list

**Comment by Gabrieldowen** [comment]
I'll get to the small fixes before I merge

**Pull request merged**